lessons/02-javascript/README.md

```diff
@@ -152,6 +157,8 @@ for (var key in object) {
 }
 ```
 
+Note that the order of iteration is not guarenteed with this syntax and later, if you use libraries which modify the prototype (more on this later) of base types, can have strange behavior.
```

Comment:
- guarAnteed
- ...with this syntax. Later, if you use libraries which modify the prototype ... of base types, you may encounter strange behavior. These strange behaviors can be solved using the following syntax:
```node
array.forEach(function(element) {
  console.log(element);
  ...
}

Object.keys(object).forEach(function(key) {
  var value = object[key];
  ...
}
```